lib/core/src/bindings.rs

```diff
@@ -107,15 +107,15 @@ impl BindingLiquidSdk {
 
     pub async fn send_payment(
         &self,
-        req: PrepareSendResponse,
+        req: SendPaymentRequest,
     ) -> Result<SendPaymentResponse, PaymentError> {
         self.sdk.send_payment(&req).await
     }
 
     pub async fn prepare_receive_payment(
         &self,
-        req: PrepareReceivePaymentRequest,
-    ) -> Result<PrepareReceivePaymentResponse, PaymentError> {
+        req: PrepareReceiveRequest,
+    ) -> Result<PrepareReceiveResponse, PaymentError> {
```

Comment:
Why were these renamed? Doesn't this go against the convention of "method name + Request/Response"?

Reply:
Previously the methods were named `PrepareReceivePaymentReq/Res` and `PrepareReceiveOnchainReq/Res`, but the distinction is now not necessary as the two have been merged. This is now the same as send, where it's simply `PrepareSendReq/Res`